dbms/src/Columns/ColumnAggregateFunction.cpp

```diff
@@ -33,6 +35,23 @@ void ColumnAggregateFunction::addArena(ArenaPtr arena_)
     arenas.push_back(arena_);
 }
 
+bool ColumnAggregateFunction::convertion(MutableColumnPtr *res_) const
```

Comment:
Name is not good (difficult to understand, what's happening). Also comment to function is needed.

Reply:
This function is used only in two function below and helps to avoid repetition. Have no idea of a better naming

Reply:
Maybe `tryFinalizeAggregateFunction`